modules/statistics/CallStats.js

```diff
@@ -376,7 +376,7 @@ export default class CallStats {
                 const match = options.confID.match(/.*\/(.*)\/.*/);
 
                 // if there is no tenant, we will just set '/'
-                configParams.siteID = (match && match[1]) || '/';
+                configParams.siteID = options.siteID || (match && match[1]) || '/';
```

Comment:
The way it is, siteID will not be used if confID is missing ... I was wondering whether we should separate them, is siteID is passed use it, otherwise if confID is passed extract it from confID, otherwise fallback to `/`

Reply:
Hmm it looked like it was meant to be a package deal - you either get both or nothing. We might get siteID and no confID this way, is that supported?

Reply:
Yeah it is not mentioned in the doc that confID is optional, you are right. If siteID is optional mark it as such in the doc?

Reply:
Good point - marked as optional